WNPRC_Purchasing/resources/queries/ehr_purchasing/myOpenRequests.sql

```diff
@@ -0,0 +1,20 @@
+-- open requests assigned to the current purchase admin user
+SELECT
+    pr.rowId AS requestNum,
+    pr.created   AS requestDate,
+    pr.vendorId,
+    items.totalCost,
+    pr.account || pr.otherAcctAndInves AS account,
+    pr.qcState   AS requestStatus,
+    pr.createdBy AS requester,
+    pr.assignedTo
+FROM ehr_purchasing.purchasingRequests pr
+         LEFT JOIN
+     (
+         SELECT requestRowId,
+                round(sum(quantity * unitCost), 2) AS totalCost
```

Comment:
I didn't catch this earlier, but we should make ehr_purchasing.lineItems.unitCost a NUMERIC(10,2) instead of a DOUBLE. Using floating point numbers for currency is problematic because of rounding problems like this.

Reply:
Also, can we inject a virtual column on purchasingRequest that does this sum instead of having to do it in a bunch of separate queries?

Reply:
done